odoo/orm/registry.py

```diff
@@ -758,6 +780,10 @@ def check_tables_exist(self, cr: Cursor) -> None:
             for table in missing_tables:
                 _logger.error("Model %s has no table.", table2model[table])
 
+    @property
+    def null_constraints_validated(self) -> bool:
```

Comment:
Since `self._null_constraints_validated` is inited as `False` and can only be set to `True` in `check_null_constraints` which is very close to the end of `load_modules`, can we directly use `self._null_constraints_validated` and make it public?